shared/team-building/index.tsx

```diff
@@ -413,7 +414,11 @@ class TeamBuilding extends React.PureComponent<Props, {}> {
             stickySectionHeadersEnabled={false}
             selectedIndex={Styles.isMobile ? undefined : this.props.highlightedIndex || 0}
             sections={this.props.recommendations}
-            keyExtractor={(item: SearchResult | ImportContactsEntry) => {
+            keyExtractor={(item: SearchResult | ImportContactsEntry, index: number) => {
+              if (!isImportContactsEntry(item) && item.contact) {
+                // Ids for contacts are not guaranteed to be unique
+                return item.userId + index
```

Comment:
why not key all rows like this?

Reply:
(except for "Import contacts" row)

Reply:
there's at least a chance that a recommendation row will be reused as a search result, and don't want to disqualify that from happening